machinelearning/tensorflow.md

```diff
@@ -277,6 +277,44 @@ bazel build --verbose_failures -s --config=mkl_aarch64  //tensorflow/tools/pip_p
 pip install <wheel-TF2.12.0-py3.8-aarch64/*.whl>
 ```
 
+# Building TensorFlow Java from sources
+
+This section outlines the recommended way to compile TensorFlow Java from source.
+
+Note: TensorFlow jar distribution for aarch64 linux platform is blocked on CI support for [tensorflow/java](https://github.com/tensorflow/java) repo.
+
+```
+sudo apt-get install pkg-config ccache clang ant python3-pip swig git file wget unzip tar bzip2 gzip patch autoconf-archive autogen automake make cmake libtool bison flex perl nasm curl gfortran libasound2-dev freeglut3-dev libgtk2.0-dev libusb-dev zlib1g libffi-dev libbz2-dev zlib1g-dev
+
+sudo apt install maven default-jdk
+
+# Install bazel for aarch64
+# Bazel version required depends on the TensorFlow version, please install the correct one
+# https://github.com/tensorflow/tensorflow/blob/master/.bazelversion captures the bazel version details
+mkdir bazel
+cd bazel
+wget https://github.com/bazelbuild/bazel/releases/download/5.1.1/bazel-5.1.1-linux-arm64
+mv bazel-5.1.1-linux-arm64 bazel
+chmod a+x bazel
+export PATH=/home/ubuntu/bazel/:$PATH
+
+# Build and install javacpp-presets.
+# Clone the following forked repo to exclude the libraries that are not supported and not required
+git clone https://github.com/snadampal/javacpp-presets.git
+cd javacpp-presets
+git checkout tfjava_aarch64
+mvn install -Djavacpp.platform=linux-arm64 -Dmaven.javadoc.skip=true -X -T 16
+
+# Build and install java
```

Comment:
Is this a Tensorflow flavor of the JVM or a Tensorflow package written in Java?

Reply:
it's the Java bindings for TensorFlow,  i simply used the repo name in the comment, but i understand the confusion, will edit the comment.

Reply:
done. updated the comment to make it more explicit.